Source/ApplicationServices/SearchEngineCommands.cs

```diff
@@ -43,28 +43,18 @@ public static void UpdateBooks(IEnumerable<Book> books)
 			else
 			{
 				foreach (var book in books)
-				{
-					UpdateLiberatedStatus(book);
-					UpdateBookTags(book);
-					UpdateUserRatings(book);
-				}
+					UpdateUserDefinedItems(book);
 			}
 		}
 
-		public static void FullReIndex() => performSafeCommand(e =>
-			fullReIndex(e)
-		);
-
-		internal static void UpdateLiberatedStatus(Book book) => performSafeCommand(e =>
-			e.UpdateLiberatedStatus(book)
-		);
-
-		internal static void UpdateBookTags(Book book) => performSafeCommand(e =>
-			e.UpdateTags(book.AudibleProductId, book.UserDefinedItem.Tags)
-		);
+		public static void FullReIndex() => performSafeCommand(fullReIndex);
```

Comment:
This is the kind of refactor where, after you see it, you say -- why the hell did I ever do it my way in the first place? ::forehead-slap::